mantle/platform/qemu.go

```diff
@@ -777,7 +794,7 @@ func (builder *QemuBuilder) Exec() (*QemuInstance, error) {
 			return nil, errors.Wrapf(err, "qemu estimating processors")
 		}
 		// cap qemu smp at some reasonable level; sometimes our tooling runs
-		// on 32-core servers (64 hyperthreads) and there's no reason to
+		// on 32-core oservers (64 hyperthreads) and there's no reason to
```

Comment:
Typo?  Or, as I understand it in Japanese, one can [prefix things as an honorific with "o"](https://www.japantimes.co.jp/life/2015/04/06/language/must-knows-nos-s-s-go/#.XoZfVukpCt8) and I guess 64 core servers are kind of worthy of an honorific so maybe you meant that?

Reply:
Ha. Typo. :)